pkgs/development/python-modules/pycatch22/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, pandas
+}:
+
+buildPythonPackage rec {
+  pname = "pycatch22";
+  version = "0.4.3";
+  format = "pyproject";
```

nits:
```suggestion
  pyproject = true;
```
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters